### PR TITLE
Search Functionality - Search recipes

### PR DESCRIPTION
**Pull request by tochman**
### https://www.pivotaltracker.com/story/show/157359973
-------------------
* adds basic search functionality using Chewy and Elasticsearch

### CODE REVIEW

**Review by GeorgeJLewis — APPROVED** [review]
Looks good.

**Comment by tochman** [comment]
I would merge this one now. Please request a review from a coach that did not work on this one.

File: Gemfile [hunk 1]
```diff
@@ -14,6 +14,7 @@ gem 'haml-rails', '~> 1.0'
 gem 'devise'
 gem 'omniauth-facebook', '~> 4.0'
 gem 'pundit'
+gem 'chewy'
```
Comment (faraznaeem) [inline Gemfile:17]: What does Chewy do?

Comment (magnus-thor) [reply Gemfile:17]: Chewy is an ODM and wrapper for the official Elasticsearch client and it provides functionality for Elasticsearch index handling, documents import mappings and chainable query DSL

**Review by faraznaeem — APPROVED** [review]

**Pull request merged**